File: Python Exams/Basic Exam Collection/Basics Online Exam - 20 and 21 April 2019/02. Easter Party/02. Easter Party.py
guests_number = int(input())
invite_price = float(input())
budget = float(input())

if 10 <= guests_number <= 15:
    invite_price *= 0.85
elif 15 < guests_number <= 20:
    invite_price *=  0.80
elif guests_number > 20:
    invite_price *= 0.75

cake_price = budget * 0.10

total_sum = guests_number * invite_price + cake_price

if budget < total_sum:
    print(f'No party! {total_sum - budget:.2f} leva needed.')
else:
    print(f'It is party time! {budget - total_sum:.2f} leva left.')

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 20 and 21 April 2019/02. Easter Party/02. Easter Party.js
 function easterParty(guestsNumber, invitePrice, budget) {

     if (10 <= guestsNumber && guestsNumber <= 15) {
         invitePrice *= 0.85
     } else if (15 < guestsNumber && guestsNumber <= 20) {
         invitePrice *= 0.80
     } else if (guestsNumber > 20)
         invitePrice *= 0.75

     let cakePrice = budget * 0.10
     let totalSum = guestsNumber * invitePrice + cakePrice

     if (budget < totalSum)
         return `No party! ${(totalSum - budget).toFixed(2)} leva needed.`
     return `It is party time! ${(budget - totalSum).toFixed(2)} leva left.`
 }

 //console.log(easterParty(18, 30, 450))